src/components/dashboard/FeedList.js

```diff
@@ -127,13 +128,13 @@ const FeedList = ({
                 dialogShown: true,
               })
               userStorage.updateOTPLEventStatus(id, 'pending')
-              showErrorDialog('The payment could not be canceled at this time', 'CANCEL-PAYMNET-1')
+              showErrorDialog('The payment could not be canceled at this time. Please try again', ExceptionCode.E11)
             })
           } catch (e) {
             log.error('cancel payment failed - quick actions', e.message, e, { dialogShown: true })
             canceledFeeds.current.pop()
             userStorage.updateOTPLEventStatus(id, 'pending')
-            showErrorDialog('The payment could not be canceled at this time', 'CANCEL-PAYMNET-2')
+            showErrorDialog('The payment could not be canceled at this time. Please try again', ExceptionCode.E13)
```

Comment:
add decorate

Reply:
it already contains understandable error message